airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/token.py

```diff
@@ -115,3 +119,103 @@ def token(self) -> str:
         auth_string = f"{self._username.eval(self.config)}:{self._password.eval(self.config)}".encode("utf8")
         b64_encoded = base64.b64encode(auth_string).decode("utf8")
         return f"Basic {b64_encoded}"
+
+
+cacheSessionTokenAuthenticator = TTLCache(maxsize=1000, ttl=86400)
```

Comment:
can you add a comment on how those values were chosen for posterity? Is it worth parameterizing them?

Reply:
done